lib/bolt/node/winrm.rb

```diff
@@ -38,7 +38,7 @@ def disconnect
     end
 
     def shell_init
-      return Bolt::Node::Success.new if @shell_initialized
+      return nil if @shell_initialized
```

Comment:
Should we merge BOLT-72 before this? Seems like they might overlap.

Reply:
Yes, please merge #153 first

Reply:
It looks like you did that?

Reply:
Yes, done.